pkgs/os-specific/linux/iwd/default.nix

```diff
@@ -32,9 +32,6 @@ in stdenv.mkDerivation rec {
     python3Packages.pygobject3
   ];
 
-  # Enable when it works again
-  enableParallelBuilding = false;
-
   configureFlags = [
     "--with-dbus-datadir=$(out)/etc/"
     "--with-dbus-busdir=$(out)/usr/share/dbus-1/system-services/"
```

Comment:
Can you drop the `usr` here? It should not be there.

Reply:
Also maybe use `placeholder`.

Reply:
Fixed, sorry for delay :).